Colours_And_Fonts_index_v1.html

```diff
@@ -88,25 +86,57 @@ <h2>Information</h2>
 
 			<br>Students will be required to wear appropriate mufti clothing on all days. This involves good walking or sports shoes, a raincoat, sunblock, and suitably warm clothing.<br>
 
-			<br>Students will not be allowed to participate in these trips if they do not have the correct gear. If you have any difficulty supplying these required items, please indicate on the return form, and we will endeavour to assist where possible. On all trips, students will be expected to 				abide by the school rules. This includes absolutely no cigarettes, drugs, or alcohol. Students will be expected to conduct themselves in a manner that is respectful to both the supervising teachers and other students. Anyone who does not comply with these requirements will be sent back 				to school and/or home at the expense of their parents or carers.<br>
+			<br>Students will not be allowed to participate in these trips if they do not have the correct gear. If you have any difficulty supplying these required items, please indicate on the return form, and we will endeavour to assist where possible. <br>On all trips, students will be expected to 				abide by the school rules. This includes absolutely no cigarettes, drugs, or alcohol. Students will be expected to conduct themselves in a manner that is respectful to both the <br>supervising teachers and other students. Anyone who does not comply with these requirements will be sent back 				to school and/or home at the expense of their parents or carers.<br>

```

Comment:
I added break <br> in my text so the text doesn't go off my content (css grid) as it kept going out so I had to add multiple breaks so the text doesn't come off the page.